content/swift/concepts/sets/terms/subtracting/subtracting.md

```diff
@@ -0,0 +1,38 @@
+---
+Title: '.subtracting()'
+Description: 'Returns a new set containing the elements from the target set that are NOT in the given set.'
+  - 'Mobile Development'
+  - 'Computer Science'
+Tags:
+  - 'Data Types'
+  - 'Variables'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`.subtracting()`** method creates a new set containing the elements from the target set that are NOT in the given set.
+
+## Syntax
+
+```pseudo
+newSet = targetSet.subtracting(givenSet)
+```
```

Comment:
Generally we have a sentence explaining the syntax snippet. i.e. "The elements of `givenSet` are removed from the elements of `targetSet` and what elements remain are placed into `newSet`."